engine/equations.cc

```diff
@@ -760,7 +760,25 @@ namespace MathDAG
           }
       }
     return r;
-  }     
+  }
+  
+  VariableDAGPtr SystemOfEquations::getNodeFromVar(const VariablePtr v)
+  {
+    NodePtr r;
+    if (expressionCache.exists(*v))
```

Comment:
From what I remember, integral inputs occupy a different "namespace" within the expressionCache. Need to look at the code...